kapre/augmentation.py

```diff
@@ -0,0 +1,85 @@
+import tensorflow as tf
+from tensorflow.keras import backend as K
+from . import backend
+from .backend import _CH_FIRST_STR, _CH_LAST_STR, _CH_DEFAULT_STR
+from tensorflow.python.ops import array_ops
+from tensorflow.keras.layers import Layer
+from tensorflow.python.keras.utils import tf_utils
+
+__all__ = [
+    'SpecAugment'
+]
+
+class SpecAugment(Layer):
+    """
+    # `SpecAugment`
+
+    ```python
+    kapre.augmentation.SpecAugment(time_param=10, freq_param=10, **kwargs)
+    ```
+
+    Add masking to input data and output it
+    Args:
+        x: Spectrogram
+        freq_param: Param of freq masking
+        time_param: Param of Time masking
+    Returns: 
+        Masked Tensor of Spectrogram
+    """
+    def __init__(
+        self, 
+        freq_param = None, 
+        time_param = None,
+        input_data_format:str='default',
+        **kwargs
+    ):
+        
+        super(SpecAugment, self).__init__(**kwargs)
+
+        backend.validate_data_format_str(input_data_format)
+
+        self.freq_param = freq_param
+        self.time_param = time_param
+        self.uses_learning_phase = True
+        self.supports_masking = True
+
+        assert (freq_param is not None) or (time_param is not None), "at least one param value should be defined"
```

Comment:
Let's use `if: raise()` instead of `assert`! That's the new way I'm following recently (for a good reason)